newdle/client/src/components/creation/userSearch/UserSearch.js

```diff
@@ -71,7 +71,12 @@ export default function UserSearch() {
           {participants.length !== 0 ? (
             <List selection relaxed>
               {participants.map(participant => (
-                <List.Item key={participant.email} className={styles['participant-list-item']}>
+                <List.Item
+                  // when creating a new newdle, all participants have an email, but when editing
+                  // we may also have participants without one - but everyone has an ID there
+                  key={participant.email || participant.id}
```

Comment:
We could probably just use `id` for everyone instead, right?

Reply:
we don't have one for newly added participants who aren't in the db yet

Reply:
I was just going to check that now too, ok sounds good!